tests/foreman/cli/test_repository.py

```diff
@@ -101,7 +101,7 @@ def test_positive_create_1(self):
                 self.assertEqual(new_repo['name'], name)
 
     @tier1
-    def test_positive_create_2(self):
+    def test_positive_create_name_label(self):
```

Comment:
`test_positive_create_with_name_label` or `test_positive_create_with_name_and_label`(I prefer more second option, however first is more correct in terms of conventions)